common-npm-packages/artifacts-common/package.json

```diff
@@ -13,14 +13,14 @@
   "license": "MIT",
   "dependencies": {
     "@types/fs-extra": "8.0.0",
-    "@types/node": "10.12.9",
+    "@types/node": "^10.17.0",
```

Comment:
Can you remove the caret (I hope thats what its called). This would update the version to 11 if available as well right?

Reply:
that is what the [guidance](https://dev.azure.com/mseng/AzureDevOps/_wiki/wikis/AzureDevOps.wiki/18197/Upgrading-tasks-to-Node-10-(previously-Node-14)) suggested.  If you still disagree, I can remove it.

Reply:
I think it's better to leave it out, to avoid any "automatic" upgrading to node and cause any issues under the radar initially. I am not sure tho

Reply:
Yep, this will update it to the next version if its available, and there be dragons ...

Reply:
Technically a caret (until now I always imagined it's spelled carrot) means that we accept only non-breaking changes (no major version changes). So we'll get any update < 11.x.x. Only minor or patch versions can change which means there shouldn't be dragons (right?). https://docs.npmjs.com/cli/v6/using-npm/semver#caret-ranges-123-025-004

Reply:
^1.2.3 := >=1.2.3 <2.0.0
So I'm fine going with as is but I'l leave the final decision to @shadargee1982